fix(my-account): add concepts (#82)
scaleway/docs-content: compute/bare-metal-cloud/concepts.mdx

## Bare Metal Server

Bare Metal servers allow you to order dedicated servers on-demand, like cloud Instances (Bare Metal as-a-Service). Bare Metal servers can be used for large workloads, big data and applications that require increased security and dedicated resources.

</Concept>

<Concept opened>

## Flexible IP

Flexible IP addresses are public IP addresses which you can hold independently of any Instance. When you create a Scaleway Instance, its public IP address is also a flexible IP address. Flexible IP addresses can be attached to and detached from any of your Instances as you wish. You can keep a number of flexible IP addresses in your account at any given time. When you delete a flexible IP address, it is disassociated from your account to be used by other users.

</Concept>

<Concept>

## Image

> An image is a file that contains the OS, application data, and any files that might be related to your applications. If you don’t know yet which image you want to run on the Bare Metal, you can order the server without preinstalled image.

Reviewer: ```suggestion
An image is a file that contains the OS, application data, and any files that might be related to your applications. If you do not know yet which image you want to run on the Bare Metal, you can order the server without preinstalled image.
```
